webpack.config.js

```diff
@@ -73,6 +73,16 @@ module.exports = {
         hot: true,
         inline: true,
         progress: true,
-	colors: true
-    }
+	    colors: true
+    },
+    devtool: (process.env.NODE_ENV !== 'production') ? 'source-map' : '',
```

Comment:
I think you had a bad merge or rebase here... this is a duplicate line/key in the dict.